docs/src/examples/minibatch.md

```diff
@@ -0,0 +1,81 @@
+# Training a Neural Ordinary Differntial Equation with Mini-Batching
+
+When training universal differential equations it is often helpful to batch our data. This is particularly useful when working with large sets of training data. Let us take a look at how this works with the Lotka-Volterra equation. 
```

Comment:
we're moving towards a style where there's a copy-pastable code at the top, and then a version with interspersed explanation below it.